lib/atscppapi/examples/Makefile.am

```diff
@@ -15,6 +15,8 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+SUBDIRS = experimental
```

Comment:
I don't think that we need experimental example plugins. Just make this an example.